common/baselines/tedlium2/gmm/baseline_args.py

```diff
@@ -320,10 +320,10 @@ def get_sat_args():
         "feature_cache": "mfcc",
         "cache_regex": "^mfcc.*$",
         "cmllr_mixtures": "estimate_mixtures_sdm.tri",
-        "iters": [8, 10],
+        "iters": [8, 9, 10],
         "feature_flow": "uncached_mfcc+context+lda",
-        "pronunciation_scales": [1.0],
-        "lm_scales": [25],
+        "pronunciation_scales": [0.0],
+        "lm_scales": [25, 20, 8.0],
```

Comment:
I don't recall how we came up with these values, maybe they are off, but currently vtln+sat with lm_scale either 20 or 25 is best.

Reply:
```suggestion
        "lm_scales": [8.0, 20.0, 25.0],
```

Reply:
> I find 25.0 just unrealistically high even for a CART hybrid model

In contrast to NN based models, the GMM-based models (and yes, it is semantically ok here to have the word "model" twice) need a higher LM scale. I have seen Zoltans Tandem-GMMs go up to a LM scale of 100.

For (CART or factored) hybrid DNN-HMM based models I agree that 25.0 would be too large in general and a range around ~10.0  should be explored.

> We need float values here.

+1 from my side. the values should be float just so that NN-kiddies in the future get reminded that non-integer values are also possible in principle.